instance/static/html/instance/appserver.html

```diff
@@ -79,8 +79,12 @@ <h4>{{ appserver.name }}</h4>
           <td>{{ instance.owner_organization }}</td>
         </tr>
         <tr>
-          <td>configuration_source_repo_url</td>
-          <td>{{ instance.configuration_source_repo_url }}</td>
+          <td>Owner organization</td>
+          <td>{{ instance.owner_organization }}</td>
+        </tr>
```

Comment:
@ahmedaljazzar This info is now duplicated (cf. L77-L80 above), please remove one of two occurrences.